redshells/model/graph_convolutional_matrix_completion.py

```diff
@@ -339,6 +344,26 @@ def _get_user_feature(cls, user_ids: List, item_ids: List, with_user_embedding,
             user_feature = session.run(graph.user_encoder, feed_dict=feed_dict)
         return user_feature
 
+    @classmethod
+    def _get_item_embedding(cls, user_ids: List, item_ids: List, with_user_embedding,
```

Comment:
Please rename this to `_get_item_feature` because there is `_get_user_feature `.

Reply:
I think this function is almost same with `_get_user_feature `, please implement functions to make codes simple.

Reply:
combined